### DISCUSSION TITLE
中国区用户 ios无法下载

### DISCUSSION THREAD

**Discussion by ZKunZhang** [community/community#4488] [Mobile] [answered]
为什么中国区用户无法下载此软件

**Comment by qinyihao** [comment]
> 为什么中国区用户无法下载此软件

Translation: Why can’t I download the GitHub app in the App Store in China?

**Reply by ZKunZhang** [reply]
yes Ios can't dowload githubApp in China

**Reply by qinyihao** [reply, marked as answer]
Yes, I know. That's because the Chinese government doesn't allow that kind of app to be downloaded in China due to some sensitive reasons.

**Reply by LeaFrock** [reply]
Any workaround？
